.eslintrc.json

```diff
@@ -36,22 +38,13 @@
     }
   ],
   "rules": {
-    "no-extra-parens": [
-      "error",
-      "functions"
-    ],
-    "no-inner-declarations": 0,
-    "consistent-return": 0,
+    "prettier/prettier": "error",
```

Comment:
the prettier settings (ignore and config) should ideally be specified here, so that prettier is only run via eslint.

Reply:
I don't think that's doable. Also, why do you want to enforce that it's only run via eslint? It shouldn't matter how someone wants to run prettier as long as they use the project's settings.